src/views/prompt-view.js

```diff
@@ -24,19 +24,21 @@ const View = require('./view');
  */
 class PromptView extends View {
   // eslint-disable-next-line require-jsdoc
-  render({ matchedEntities, missingEntities }) {
-    logger.debug('render', { matchedEntities, missingEntities });
-    return this.renderEntities(matchedEntities, missingEntities);
+  render({ matchedEntities, missingEntities, ...dialogData }) {
```

Comment:
@KevinDepuydt why spread here?
`dialogData` can be anything, from object to primitive so this will break if user returns anything other than an array or object in `dialogWillDisplay` I think?

Reply:
My bad, I just changed this !